third_party/move/move-analyzer/src/symbols.rs

```diff
@@ -1,6 +1,9 @@
 // Copyright (c) The Move Contributors
 // SPDX-License-Identifier: Apache-2.0
 
+#![allow(clippy::unwrap_or_default)]
+#![allow(clippy::non_canonical_partial_ord_impl)]
```

Comment:
Any ones of note?

Reply:
Both of these are used many times throughout the file. Seemed cleaner to allow them for the entire file 😄